_posts/salads/2019-04-28-spicy-tofu-celery-salad.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Backpacking Duck Confit Salad
+category: salads
+tags: recipeish, potluck
+---
+This recipe is sized for a potluck!
+
+*Ingredients & Sourcing*
+- 4 blocks Tofu Gan -- This is the center piece of the dish. You can find it labeled as "Braised Tofu" or "Tofu Gan" or "Pressed Tofu Blocks" in the store. They're very firm, they're brown on the outside, and do not come in water (they come in plastic though). Hodo Soy calls theirs "Braised tofu", and that's widely available where I live. 
```

Comment:
I think you could denote the sauce ingredients with a sublist or as a separate list? 

Like, 
```
*You'll need:*
... <tofu and celery here>

*And for the sauce:*
```

Also, I think a brief description at the very beginning conveying something about this dish would provide helpful context.